### PR TITLE
Student exchange

### PR DESCRIPTION
**Pull request by Gordoburrito**
image

# Description

# Links

Related pull requests:

Design:

# Additional Info

### CODE REVIEW

File: src/pages/StudentExchange/StudentExchange.js [hunk 1]
```diff
@@ -4,13 +4,18 @@ import Layout from 'components/Layout/Layout';
 const StudentExchange = ({ data }) => {
   if (!data) return null;
   else {
-    const { title, body } = data.student_exchange_page;
+    // console.log(data.builder)
```
Comment (atomictangerine) [inline src/pages/StudentExchange/StudentExchange.js:7]: forgotten console log

File: src/pages/Events/Events.js [hunk 2]
```diff
@@ -18,16 +19,16 @@ const Events = () => {
         <h4 className="events__subtitle">Upcoming Events</h4>
         <div className="events__cards">
           { events.map((event, i) =>
-            !event.acf.annual ? <CardEvent key={ i } props={ event } /> : ''
+            today < new Date(event.acf.event_date) ? <CardEvent key={ i } props={ event } /> : ''
```
Comment (atomictangerine) [inline src/pages/Events/Events.js:22]: what happens when the event date and today's date are the same?

File: src/pages/StudentExchange/StudentExchange.js [hunk 3]
```diff
@@ -4,13 +4,18 @@ import Layout from 'components/Layout/Layout';
...
+    let { builder } = data;
     return (
       <Layout>
-        <div className="studentexchange__container">
-          <h2 position="left" className="studentexchange__title">{title}</h2>
-          <div className="studentexchange__description" dangerouslySetInnerHTML={{__html: body}} />
-        </div>
+        { builder.map((el, i) =>
+            <div className="studentexchange__container" key={i}>
```
Comment (atomictangerine) [inline src/pages/StudentExchange/StudentExchange.js:12]: using index as a key when doing a map is [a react anti-pattern](https://medium.com/wesionary-team/using-index-as-a-key-is-an-anti-pattern-in-react-8e5db3aea3a9)

**Comment by jekumura** [comment]
@Gordoburrito Would you be able to review & correct the above comments?

File: src/pages/StudentExchange/StudentExchange.js (hunk 3 above)
Comment (Gordoburrito) [reply src/pages/StudentExchange/StudentExchange.js]: Updated to element's title instead

**Review by jekumura — APPROVED** [review]

**Pull request merged**